scripts/bump-wrapper-sdk-version.sh

```diff
@@ -0,0 +1,101 @@
+            newWrapperSdkVersion=$VALUE
+            ;;
+        --oldAndroidVersionCode)
+            oldAndroidVersionCode=$VALUE
+            ;;
+        --newAndroidVersionCode)
+            newAndroidVersionCode=$VALUE
+            ;;
+        *)
+    esac
+    shift
+done
+
+# Exit if any of the parameters have not been set
+if [ -z $oldWrapperSdkVersion ]; then
+    echo "oldWrapperSdkVersion cannot be empty"
+    exit 1
+fi
+if [ -z $newWrapperSdkVersion ]; then
+    echo "newWrapperSdkVersion cannot be empty"
+    exit 1
+fi
+if [ -z $oldAndroidVersionCode ]; then
+    echo "oldAndroidVersionCode cannot be empty"
+    exit 1
+fi
+if [ -z $newAndroidVersionCode ]; then
+    echo "newAndroidVersionCode cannot be empty"
+    exit 1
+fi
+
+echo "React-Native SDK $oldWrapperSdkVersion will be updated to $newWrapperSdkVersion"
+echo "React-Native SDK Android VersionCode $oldAndroidVersionCode will be updated to $newAndroidVersionCode"
+
+# Update wrapper sdk version in package.json for mobile-center, mobile-center-crashes, mobile-center-analytics, 
+# mobile-center-push and mobile-center-link-script NPM packages
+
+export newVersion=$newWrapperSdkVersion
+cat ./mobile-center/package.json | jq -r '.version = env.newVersion' | jq -r '.dependencies."mobile-center-link-scripts" = env.newVersion' > ./mobile-center/package.json.temp && mv ./mobile-center/package.json.temp ./mobile-center/package.json
+cat ./mobile-center-crashes/package.json | jq -r '.version = env.newVersion' | jq -r '.dependencies."mobile-center" = env.newVersion' > ./mobile-center-crashes/package.json.temp && mv ./mobile-center-crashes/package.json.temp ./mobile-center-crashes/package.json 
+cat ./mobile-center-analytics/package.json | jq -r '.version = env.newVersion' | jq -r '.dependencies."mobile-center" = env.newVersion' > ./mobile-center-analytics/package.json.temp && mv ./mobile-center-analytics/package.json.temp ./mobile-center-analytics/package.json
+cat ./mobile-center-push/package.json | jq -r '.version = env.newVersion' | jq -r '.dependencies."mobile-center" = env.newVersion' > ./mobile-center-push/package.json.temp && mv ./mobile-center-push/package.json.temp ./mobile-center-push/package.json
+cat ./mobile-center-link-scripts/package.json | jq -r '.version = env.newVersion' > ./mobile-center-link-scripts/package.json.temp && mv ./mobile-center-link-scripts/package.json.temp ./mobile-center-link-scripts/package.json
+
+# Update wrapperk sdk version and android VersionCode in Android build.gradle for mobile-center, mobile-center-crashes, mobile-center-analytics,
+# mobile-center-push and RNMobileCenterShared projects
+
+gradleFileContent="$(cat ./mobile-center/android/build.gradle)"
+gradleFileContent=`echo "${gradleFileContent/versionName \"$oldWrapperSdkVersion\"/versionName \"$newWrapperSdkVersion\"}"`
+echo "${gradleFileContent/versionCode $oldAndroidVersionCode/versionCode $newAndroidVersionCode}" > ./mobile-center/android/build.gradle
```

Comment:
This technic of string replacement feels a bit dangerous to me, it will replace the first occurrence of the version in the file but what if the string representation of this version is used elsewhere for another purpose, it might replace the wrong version.